src/main/java/io/tiledb/spark/TileDBDataReaderPartitionScan.java

```diff
@@ -597,6 +626,7 @@ private int getScalarValueColumn(String name, Datatype dataType, int index) thro
       case TILEDB_INT64:
       case TILEDB_UINT32:
       case TILEDB_UINT64:
+      case TILEDB_DATETIME_MS:
```

Comment:
Lets open a clubhouse stoy to support all the datetime datatypes

Reply:
Opened: https://app.clubhouse.io/tiledb-inc/story/6436/support-all-datetime-datatypes